src/petals/bloom/from_pretrained.py

```diff
@@ -45,6 +48,10 @@ def load_pretrained_block(
         cache_dir = DEFAULT_CACHE_DIR
 
     block = WrappedBloomBlock(config)
+    if load_in_8bit:
+        block = replace_8bit_linear(block)
+        block = block.to(device)
```

Comment:
I moved `replace_8bit_linear` here because it's not possible to correctly load the quantized Linear8bitLt checkpoint into the model before it's converted and quantized